app/models/user.rb

```diff
@@ -216,4 +203,28 @@ def self.active
     where(deactivated_at: nil)
   end
 
+  def internal?
+    price_groups.include?(PriceGroup.base)
+  end
+
+  def update_price_group!(params)
+    if params[:internal] == "true"
+      price_group_members.find_by(price_group: PriceGroup.external).try(:destroy)
+      price_group_members.create(price_group: PriceGroup.base)
```

Comment:
I feel like this would be clearer as `find_or_create_by`. I was initially confused as to why it wan't creating duplicates every time I submitted the form.